distributed/worker.py

```diff
@@ -2291,8 +2275,7 @@ async def execute(self, key: str, *, stimulus_id: str) -> StateMachineEvent:
                         self.scheduler_delay,
                     )
                 elif "ThreadPoolExecutor" in str(type(e)):
-                    result = await self.loop.run_in_executor(
-                        e,
+                    result = await offload(
```

Comment:
Please do not reuse `offload` for this. Offloading is something with a clearly attached meaning and we typically do not refer to user functions being "offloaded" but rather refer to event blocking things as being offloaded.

Reply:
Done.